services/navigation/builtin/builtin.go

```diff
@@ -303,6 +303,11 @@ func (svc *builtIn) startWaypoint(ctx context.Context, extra map[string]interfac
 		extra["motion_profile"] = "position_only"
 	}
 
+	motionCfg := motion.MotionConfiguration{
+		LinearMPerSec:     svc.metersPerSec,
+		AngularDegsPerSec: svc.degPerSec,
```

Comment:
i think this PR should be merged before

https://github.com/viamrobotics/rdk/pull/2633

since we can then tie in nav defaults to the moveonglobe call which then interacts with the defaults set in kinematics

Reply:
I agree lets do the config ticket first